index.html

```diff
@@ -0,0 +1,462 @@
+            </div>
+            <p class="card-note">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="lang-tech">
+              <li class="tech-used">HTML</li>
+              <li class="tech-used">Bootstrap</li>
+              <li class="tech-used">Ruby</li>
+            </ul>
+            <button class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <div class="background">
+          <div class="cards">
+            <div class="card-title">
+              <p>Professional Art</p>
+              <p>Printing Data</p>
+            </div>
+            <p class="card-note">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="lang-tech">
+              <li class="tech-used">HTML</li>
+              <li class="tech-used">Bootstrap</li>
+              <li class="tech-used">Ruby</li>
+            </ul>
+            <button class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <div class="background">
+          <div class="cards">
+            <div class="card-title">
+              <p>Professional Art</p>
+              <p>Printing Data</p>
+            </div>
+            <p class="card-note">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="lang-tech">
+              <li class="tech-used">HTML</li>
+              <li class="tech-used">Bootstrap</li>
+              <li class="tech-used">Ruby</li>
+            </ul>
+            <button class="see-project-btn">See Project</button>
+          </div>
+        </div>
+        <div class="background">
+          <div class="cards">
+            <div class="card-title">
+              <p>Professional Art</p>
+              <p>Printing Data</p>
+            </div>
+            <p class="card-note">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. Has been the industry's standard.
+            </p>
+            <ul class="lang-tech">
+              <li class="tech-used">HTML</li>
+              <li class="tech-used">Bootstrap</li>
+              <li class="tech-used">Ruby</li>
+            </ul>
+            <button class="see-project-btn">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="container2">
+        <div class="background1">
+          <button class="see-project-btn1-desk">See Project</button>
+
+          <!-- <ul class="lang-tech">
+            <li class="tech-used">HTML</li>
+            <li class="tech-used">Bootstrap</li>
+            <li class="tech-used">Ruby</li>
+          </ul> -->
```

Comment:
- [ ] For clean code and best practices reasons, kindly make sure you remove all commented and unused code lines from your code base